backend/utils/airtable/utils.js

```diff
@@ -122,7 +124,11 @@ async function updateAirtableEntry(recordId, proposal, grantCompleted = false) {
     update["Minimum USD Requested"] = proposal.minUsdRequested;
 
   if (proposal.oneLiner) update["One Liner"] = proposal.oneLiner;
-  await base("Proposals").update(recordId, update);
+    base("Proposals").update(recordId, update).then((resp) => {
```

Comment:
Please revert this part, it was fixed by https://github.com/oceanprotocol/oceandao-proposal-portal/pull/122